tests/tests.go

```diff
@@ -234,3 +234,15 @@ func buildImage(o *option.Option, imageName string) {
 	ginkgo.DeferCleanup(os.RemoveAll, buildContext)
 	command.Run(o, "build", "-q", "-t", imageName, buildContext)
 }
+
+func buildCacheShouldNotExist() error {
+	var buildctlCmd []string
+	buildctlCmd = append(buildctlCmd, "buildctl")
+	buildctlOpts, err := option.New(buildctlCmd)
```

Comment:
nit: this is a little cleaner
```suggestion
	buildctlOpt, err := option.New([]string{"buildctl"})
```

Reply:
sure